### PR TITLE
Add fonts 

### PR DESCRIPTION
**Pull request by rafaelferreira6**
- Add Sans font;
- Add Serif font;
- Add gray dark color;
- Add typography config.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -47,6 +47,12 @@ module.exports = {
   },
 
   theme: {
+    fontFamily: {
+      "title-rg": ["Sans-Regular"],
```
Comment (jfranciscosousa) [inline tailwind.config.js]: We should have the following families:
- edgy 
- sans
- sans-regular
- sans-subtitle
- serif

**Comment by jfranciscosousa** [comment]
All of the fonts should be under `src/static/fonts` and on the CSS you reference them as `/fonts/<name of file>`

File: tailwind.config.js (hunk 1 above)
Comment (jfranciscosousa) [reply tailwind.config.js]: It's the nomenclature we are using already

Comment (rafaelferreira6) [reply tailwind.config.js]: We'll use

      edgy: ["StudioFeixenSans-Regular"],
      "edgy-medium": ["StudioFeixenSans-Medium"],
      sans: ["StudioFeixenSans-Regular-text"],
      serif: ["RecifeText-Regular"]

Comment (jfranciscosousa) [reply tailwind.config.js]: Nice, thanks

**Pull request merged**